shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/MealsSDK.kt

```diff
@@ -16,6 +16,10 @@ class MealsSDK {
         return api.postMeal(meal)
     }
 
+    @Throws(Exception::class) suspend fun patchMeal(id: String, meal: Meal): Meal {
+        return api.patchMeal(meal, id)
+    }
```

Comment:
we can just pass through the meal here.
And then get out the id from the Meal object

Reply:
Hmm, so how should ID be passed through the httpClient fun?

Reply:
Nvm just saw your other comment ^^